### ISSUE TITLE
Docker image in hub.docker.com

### ISSUE THREAD

**Issue by iBobik** [lnbits/lnbits-legend#462]
I see this repo is automatically build and published already:
https://github.com/lnbits/lnbits-legend/blob/master/.github/workflows/on-push.yml

But installation instructions requires us to build it by ourself:
https://github.com/lnbits/lnbits-legend/blob/master/docs/guide/installation.md

Could you publish hub.docker.com path to this official image, so we can use it? Currently I see there are some images, but no way hot to see which one is official:
https://hub.docker.com/search?q=lnbits&type=image

**Comment by iBobik** [comment]
It seems https://hub.docker.com/r/lnbitsdocker/lnbits is legit, because Umbrel uses it.

**Comment by trbouma** [comment]
this image is a few months out of date. It is the one that is still being used with Umbrel. How do we get the latest update pushed so umbrel can use it?

**Comment by nolim1t** [comment]
build error

```
#12 10.40   Preparing metadata (setup.py): started
#12 10.49   Preparing metadata (setup.py): finished with status 'error'
#12 10.49   error: subprocess-exited-with-error
#12 10.49   
#12 10.49   × python setup.py egg_info did not run successfully.
#12 10.49   │ exit code: 1
#12 10.49   ╰─> [1 lines of output]
#12 10.49       'pkg-config' is required to install this package. Please see the README for details.
#12 10.49       [end of output]
```

**Comment by nolim1t** [comment]
Seems that the build error affects arm64 (can't build it on my M1 which is as of arm64 architecture). 

Will try it on my arm64 linux laptop too

**Comment by iBobik** [comment]
I see there is also action to auto build and push image, but it is not configured to run automatically.

https://github.com/lnbits/lnbits-legend/blob/main/.github/workflows/on-tag.yml

**Comment by iBobik** [comment]
Hi @louneskmt, I see you created GH action for Docker build and push. Do you know reason why it does not run for each release? Could it be enabled?

**Comment by louneskmt** [comment]
The GH Action is started on tag, if the tag is X.X.X or X.X.X-something.

**Comment by louneskmt** [comment]
Hey @iBobik! The old GH Action was building the image on push on master. LNbits changed the main branch from `master` to `main` a couple of times ago (along with the change of name), so it broke the autobuild. 

#530 has been merged 10 days ago, it fixes this and follows the new release process (tag and not commit head).

**Comment by iBobik** [comment]
> #533 has been merged 10 days ago, it fixes this and follows the new release process (tag and not commit head).

I still do not see action run nor pushed image from last weeks:
https://github.com/lnbits/lnbits-legend/actions/workflows/on-tag.yml
https://hub.docker.com/r/lnbitsdocker/lnbits

**Comment by louneskmt** [comment]
Yeah, it's because the Action runs when a new version is tagged, and there has been no new release since the merge.

**Comment by iBobik** [comment]
> Yeah, it's because the Action runs when a new version is tagged, and there has been no new release since the merge.

Latest image is 3 months ago. On GH I see releases 0.5.0, 0.6.0 and 0.7.0 from then.

**Comment by louneskmt** [comment]
Sure, but #533 has been merged only 10 days ago. It is this PR that fixes the autobuild.

**Comment by iBobik** [comment]
Aha, so you have to create tag now? Minor version with just this PR mentioned?

Honza Pobořil

**Comment by louneskmt** [comment]
On the next tag / release, the image will be built and pushed automatically, yes 😁

**Comment by bezysoftware** [comment]
Can you do a patch release to trigger the run and publish current version as docker image? Or are you planning a regular release soon?

**Comment by louneskmt** [comment]
I'm not a LNbits dev, tagging @arcbtc @callebtc.

**Comment by iBobik** [comment]
I see build and push has been triggered, but in a different repository: https://hub.docker.com/r/lnbitsdocker/lnbits-legend Old one: https://hub.docker.com/r/lnbitsdocker/lnbits

**Comment by louneskmt** [comment]
That's expected, as LNbits is now called LNbits Legend. 

The tag release `#0.8.0` has successfully triggered the build and push GH Action, and the image is available on [Docker Hub](https://hub.docker.com/r/lnbitsdocker/lnbits-legend/tags). 

I still need to test it to verify that there is no issue and that LNbits is running properly, but it should be ok.

**Comment by nolim1t** [comment]
Great looks cool let me know how you go with that image

**Comment by louneskmt** [comment]
Docker image tested and working on my mainnet Umbrel @nolim1t @iBobik. This should be good to close.
